internal/playwright/config.go

```diff
@@ -141,3 +147,40 @@ func SplitSuites(p Project) (Project, Project) {
 
 	return dockerProject, sauceProject
 }
+
+func checkSupportedBrowsers(p *Project) error {
+	supportedBrwsList := []string{"chromium", "firefox", "webkit"}
+	errMsg := "browserName: %s is not supported. List of supported browsers: %s"
+
+	if p.Playwright.Params.BrowserName != "" {
+		found := false
+		for _, browser := range supportedBrwsList {
+			if p.Playwright.Params.BrowserName == browser {
+				found = true
```

Comment:
I'd just keep it simple and return immediately, rather than dealing with additional vars (`found`) that are just crutches 💪 Same applies to the code below.

Reply:
I refactored the code a bit and now it's not necessary to have this boolean vars.